gui/app/main.qml

```diff
@@ -59,7 +60,7 @@ ApplicationWindow {
             }
 
             onMenuClicked: console.log("Menu Click")
-            onAlarmSettingsClicked: console.log("Alarm Settings Clicked")
+            onAlarmSettingsClicked: alarmNotificationBanner.enabled = true
```

Comment:
This was just for debug purpose. Are you planning to merge with this?

Reply:
Fixed